templates/Program/program_editor_navigation.html.twig

```diff
@@ -0,0 +1,12 @@
+<div class="row">
+  <div class="col">
+    {% if app.user and my_program %}
+      <div id="edit-text-navigation" class="d-none mb-5 full-screen">
+        <ul id="navigation-language-list" style="list-style-type: none; padding-left: 0px;">
```

Comment:
If possible move inline styles always to CSS classes. This way they can be optimized and minified.

Reply:
Moved styling to a new CSS class